infrastructure/stage/step-functions/interfaces.ts

```diff
@@ -11,17 +11,19 @@ export type StepFunctionsName =
   | 'pushIcav2Data'
   | 'pushS3Data'
   | 'push'
-  | 'autoPackagePush'
-  | 'autoController';
+  | 'autoController'
+  | 'autoPackage'
+  | 'autoPush';
 
 export const stepFunctionsNameList: StepFunctionsName[] = [
   'packaging',
   'presigning',
   'pushIcav2Data',
   'pushS3Data',
   'push',
-  'autoPackagePush',
   'autoController',
```

Comment:
Similar to one of my comments above, autoController to have the needsNestedSfnPermissions attributes